diff --git a/node_modules/@expo/eas-json/build/build/schema.js b/node_modules/@expo/eas-json/build/build/schema.js
index 3ddd111..c17df22 100644
--- a/node_modules/@expo/eas-json/build/build/schema.js
+++ b/node_modules/@expo/eas-json/build/build/schema.js
@@ -101,6 +101,7 @@ const IosBuildProfileSchema = PlatformBuildProfileSchema.concat(joi_1.default.ob
     simulator: joi_1.default.boolean(),
     scheme: joi_1.default.string(),
     buildConfiguration: joi_1.default.string(),
+    workingDirectory: joi_1.default.string(),
     // versions
     autoIncrement: joi_1.default.alternatives().try(joi_1.default.boolean(), joi_1.default.string().valid('version', 'buildNumber')),
 }));
diff --git a/node_modules/@expo/eas-json/build/build/types.d.ts b/node_modules/@expo/eas-json/build/build/types.d.ts
index 58e1aa8..9d00798 100644
--- a/node_modules/@expo/eas-json/build/build/types.d.ts
+++ b/node_modules/@expo/eas-json/build/build/types.d.ts
@@ -57,6 +57,7 @@ export interface IosBuildProfile extends PlatformBuildProfile {
     enterpriseProvisioning?: IosEnterpriseProvisioning;
     simulator?: boolean;
     scheme?: string;
     buildConfiguration?: string;
+    workingDirectory?: string;
     autoIncrement?: IosVersionAutoIncrement;
 }
 export type BuildProfile<TPlatform extends Platform = Platform> = TPlatform extends Platform.ANDROID ? AndroidBuildProfile : IosBuildProfile;
