diff --git a/node_modules/@expo/config-plugins/build/ios/Paths.js b/node_modules/@expo/config-plugins/build/ios/Paths.js
index 39fb0a5..167e2cf 100644
--- a/node_modules/@expo/config-plugins/build/ios/Paths.js
+++ b/node_modules/@expo/config-plugins/build/ios/Paths.js
@@ -75,7 +75,7 @@ function _getRequireWildcardCache(e) { if ("function" != typeof WeakMap) return
 function _interopRequireWildcard(e, r) { if (!r && e && e.__esModule) return e; if (null === e || "object" != typeof e && "function" != typeof e) return { default: e }; var t = _getRequireWildcardCache(r); if (t && t.has(e)) return t.get(e); var n = { __proto__: null }, a = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var u in e) if ("default" !== u && {}.hasOwnProperty.call(e, u)) { var i = a ? Object.getOwnPropertyDescriptor(e, u) : null; i && (i.get || i.set) ? Object.defineProperty(n, u, i) : n[u] = e[u]; } return n.default = e, t && t.set(e, n), n; }
 const ignoredPaths = ['**/@(Carthage|Pods|vendor|node_modules)/**'];
 function getAppDelegateHeaderFilePath(projectRoot) {
-  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/*/AppDelegate.h', {
+  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/**/AppDelegate.h', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
@@ -95,7 +95,7 @@ function getAppDelegateHeaderFilePath(projectRoot) {
   return using;
 }
 function getAppDelegateFilePath(projectRoot) {
-  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/*/AppDelegate.@(m|mm|swift)', {
+  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/**/AppDelegate.@(m|mm|swift)', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
@@ -115,7 +115,7 @@ function getAppDelegateFilePath(projectRoot) {
   return using;
 }
 function getAppDelegateObjcHeaderFilePath(projectRoot) {
-  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/*/AppDelegate.h', {
+  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/**/AppDelegate.h', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
@@ -135,7 +135,7 @@ function getAppDelegateObjcHeaderFilePath(projectRoot) {
   return using;
 }
 function getPodfilePath(projectRoot) {
-  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/Podfile', {
+  const [using, ...extra] = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/**/Podfile', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
@@ -187,7 +187,7 @@ function getSourceRoot(projectRoot) {
   return path().dirname(appDelegate.path);
 }
 function findSchemePaths(projectRoot) {
-  return (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/*.xcodeproj/xcshareddata/xcschemes/*.xcscheme', {
+  return (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/**/*.xcodeproj/xcshareddata/xcschemes/*.xcscheme', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
@@ -257,7 +257,7 @@ function getPBXProjectPath(projectRoot) {
   return using;
 }
 function getAllInfoPlistPaths(projectRoot) {
-  const paths = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/*/Info.plist', {
+  const paths = (0, _glob2().withSortedGlobResult)((0, _glob().globSync)('ios/**/Info.plist', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
@@ -283,7 +283,7 @@ function getInfoPlistPath(projectRoot) {
   return using;
 }
 function getAllEntitlementsPaths(projectRoot) {
-  const paths = (0, _glob().globSync)('ios/*/*.entitlements', {
+  const paths = (0, _glob().globSync)('ios/**/*.entitlements', {
     absolute: true,
     cwd: projectRoot,
     ignore: ignoredPaths
